modules/storages/app/models/storages/project_storage.rb

```diff
@@ -61,7 +61,10 @@ def automatic_management_possible?
   end
 
   def project_folder_path
-    "#{storage.group_folder}/#{project.name.tr('/', '|')} (#{project.id})/"
+    return "#{storage.group_folder}/#{project.name.tr('/', '|')} (#{project.id})/" if storage.provider_type_nextcloud?
+    return project_folder_id if storage.provider_type_one_drive?
+
+    raise 'Unknown Storage'
```

Comment:
🟢 WDYT of creating an Exception with this name and raise it? 

🍏 Also, I've been playing with the idea of who should know about how we should format project folder paths. It is clearly storage dependent, but I'm not sure I'd move this to the storage itself. Dilemmas... >.<

Reply:
yeah, I feel you. I basically did not try to solve that question, and simply replaced the code in the current place.